app/assets/javascripts/angular/directives/frame.js.erb

```diff
@@ -41,20 +41,27 @@ angular.module("Prometheus.directives").directive('inlineFrame', ["$sce", "Varia
       });
 
       function buildFrameURL(url) {
-        var parser = document.createElement('a');
-        parser.href = url;
+        var parts = url.split('?'),
+            path = parts[0],
+            query = parts[1].split('&');
```

Comment:
also, `query = parts[1].split('&')` throws an error if there is no query string (see `web-status`, e.g.)